lib/controller.js

```diff
@@ -457,6 +460,167 @@ controller.connectToNetwork = function(opts) {
     });
 };
 
+controller.update = function(opts) {
+  return new Promise(function(resolve, reject) {
+    // If the user requested a list of builds
+    if (opts.list) {
+      // Fetch those builds from the build server
+      logs.info('Fetching available builds...');
+      return updates.fetchBuildList()
+        .then(function(builds) {
+          logs.info('Latest builds:');
+          var maxBuildsListed = 10;
+          var SHAs = Object.keys(builds);
```

Comment:
With my suggested change to the builds.json file, this would become: 

```js
// The last 10 are the 10 most recent builds
var SHAs = builds.slice(-10).map(function(build) { return build.sha; });
```